zerver/openapi/zulip.yaml

```diff
@@ -1036,8 +1036,13 @@ paths:
                             - type: object
                               description: |
                                 Event sent to all users in a Zulip organization when a new
-                                user joins. Processing this event is important to being able
-                                to display basic details on other users given only their ID.
+                                user joins or when a guest user gains access to a user.
+                                Processing this event is important to being able to display
```

Comment:
Should we also describe the cases when a guest will gain access to a user?

Reply:
I think so but possibly in /help/ documentation linked to from here; see https://github.com/zulip/zulip/pull/27544#discussion_r1401579684.